src/__experimental__/components/grouped-character/grouped-character.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Store, State } from '@sambego/storybook-state';
+import { text } from '@storybook/addon-knobs';
+import GroupedCharacter from '.';
+
+const groupedCharacterStore = new Store({
+  value: ''
+});
+
+storiesOf('Experimental/GroupedCharacter', module)
+  .add('Basic', () => {
+    return (
+      <State store={ groupedCharacterStore }>
```

Comment:
you could use the `StateDecorator` instead of the `State` component

Reply:
What is the advantage in doing that?

Reply:
in my opinion - with that decorator jsx looks cleaner